apps/backend/src/auth/TechnicalReviewAuthorization.ts

```diff
@@ -159,10 +159,12 @@ export class TechnicalReviewAuthorization {
         : technicalReviewOrProposalPk.proposalPk;
 
     const isUserOfficer = this.userAuth.isUserOfficer(agent);
-    const isFapChairOrSecOfFap =
```

Comment:
You have changed the variable name, but the function stays the same? Did you mean to just check if they are a FAP sec?